File: Pharmacity/DTOs/CheckoutDto.cs
namespace Pharmacity.DTOs
{
    public class CheckoutDto
    {
        public int MaDC { get; set; } // ID địa chỉ người dùng chọn
        public string PhuongThucTT { get; set; } // "cod", "momo", ...

        public List<int>? SelectedMalos { get; set; } // Thêm dòng này
    }
}

File: flutter/app512/lib/data/models/order/checkout_dto.dart
class CheckoutDto {
  final int maDC;
  final String phuongThucTT;
  final List<int> selectedMalos;

  CheckoutDto({
    required this.maDC,
    required this.phuongThucTT,
    required this.selectedMalos,
  });

  Map<String, dynamic> toJson() => {
    'maDC': maDC,
    'phuongThucTT': phuongThucTT,
    'selectedMalos': selectedMalos,
  };
}
